packages/api-request-builder/src/default-services.js

```diff
@@ -31,6 +31,16 @@ export default {
     endpoint: '/customers',
     features: [features.query, features.queryOne, features.queryExpand],
   },
+  customersPasswordToken: {
+    type: 'customers-password-token',
+    endpoint: '/customers/password-token',
```

Comment:
From [here](https://docs.commercetools.com/http-api-projects-customers.html#get-customer-by-password-token), is `password-token` part of the path or the query?